package-lock.json

```diff
@@ -25,9 +25,9 @@
       }
     },
     "@code-dot-org/remark-plugins": {
-      "version": "1.2.3",
-      "resolved": "https://registry.npmjs.org/@code-dot-org/remark-plugins/-/remark-plugins-1.2.3.tgz",
-      "integrity": "sha512-sRrWlbQVHPyLpKiEH1Ux2moSnSjyB9i29INA8CXWEcOD1ARh75gakzt3Bchp7FuvoZuWTlKJXeboJePgDze0Dw==",
+      "version": "1.2.4",
+      "resolved": "https://registry.npmjs.org/@code-dot-org/remark-plugins/-/remark-plugins-1.2.4.tgz",
+      "integrity": "sha512-texE/vSeXiAW9Dki/Xthm6leeYuGegRfZnwgSDScLHCNoah3uLWo+tEz/pm1hIgFHwPJjGV0fWsA7Sau6gglfg==",
```

Comment:
For my own education, how did you generate this hash?

Reply:
`npm install` !